python/setup.py

```diff
@@ -24,19 +24,33 @@
     print "You don't seem to have Cython installed."
     sys.exit(1)
   os.environ['ARCHFLAGS'] = ""
+
+  # Additional library search directories:
+  # - LD_LIBRARY_PATH (if present)
+  # - User local enviroment
+  library_dirs = []
+  # If LD_LIBRARY_PATH has been manually specified, add it to the
+  # library search path
+  if 'LD_LIBRARY_PATH' in os.environ:
+    library_dirs.append(os.environ['LD_LIBRARY_PATH'])
+  library_dirs.append(os.path.expanduser('~/.local/lib'))
+
+  # Additional include directories:
+  # - Numpy includes
+  # - User local enviroment
+  # - Current directory
+  include_dirs = []
+  include_dirs.append(np.get_include())
+  include_dirs.append(os.path.expanduser('~/.local/include'))
+  include_dirs.append('.')
```

Comment:
Having removed `../include/` from here causes the python build to fail on Travis where the libraries are installed to an non-standard location.  Maybe just have Travis install to `~/.local/`?